docs/book/src/quick-start.md

```diff
@@ -9,7 +9,8 @@ This Quick Start guide will cover:
 
 ## Prerequisites
 
-- [go](https://golang.org/dl/) version v1.15+ and < 1.16.
+- [go](https://golang.org/dl/) version v1.15+ and < 1.16(kubebuilder <= v3.0).   
+- [go](https://golang.org/dl/) version 1.16+(kubebuilder v3.1+).
```

Comment:
The book describes the latest kubebuilder major version, so there's no need to discuss minor versions
```suggestion 
- [go](https://golang.org/dl/) version 1.15+ and < 1.17.
```

Reply:
Thanks for the suggestion. IMO, `version 1.15+ and < 1.17` can be misleading to users. If they are using go 1.15.13 and kubebuilder v3.1, they will fail to initialize the project. What about changing the docs as follows:
- [go](https://golang.org/dl/) version v1.15+ and < 1.16(kubebuilder v3.0).
- [go](https://golang.org/dl/) version v1.16+(kubebuilder v3.1).

Reply:
Where are you seeing the go > 1.15 requirement? The range is [go 1.13 to < 1.17](https://github.com/kubernetes-sigs/kubebuilder/blob/359e8aa70c2bd307cd7c74cb2a956ef0df5f99e1/pkg/plugins/golang/go_version.go#L152-L154) but 1.15 is recommended because of k8s dependencies.

Reply:
in the quickstart https://book.kubebuilder.io/quick-start.html#prerequisites

Reply:
That should be changed to 1.15+ and < 1.17.

Reply:
Alright. Done.